python/docs/source/user_guide/pandas_on_spark/best_practices.rst

```diff
@@ -234,12 +234,12 @@ this operation should be avoided.
 See `Operations on different DataFrames <options.rst#operations-on-different-dataframes>`_ for more details.
 
 
-Use pandas APIs on Spark directly whenever possible
+Use pandas API on Spark directly whenever possible
 ---------------------------------------------------
 
-Although pandas APIs on Spark have most of the pandas-equivalent APIs, there are several APIs not implemented yet or explicitly unsupported.
+Although pandas API on Spark have most of the pandas-equivalent APIs, there are several APIs not implemented yet or explicitly unsupported.
 
-As an example, pandas APIs on Spark do not implement ``__iter__()`` to prevent users from collecting all data into the client (driver) side from the whole cluster.
+As an example, pandas API on Spark do not implement ``__iter__()`` to prevent users from collecting all data into the client (driver) side from the whole cluster.
```

Comment:
```suggestion
As an example, pandas API on Spark does not implement ``__iter__()`` to prevent users from collecting all data into the client (driver) side from the whole cluster.
```